projects/core/src/occ/adapters/product/default-occ-product-config.ts

```diff
@@ -11,7 +11,7 @@ export const defaultOccProductConfig: OccConfig = {
           list:
             'products/${productCode}?fields=code,name,summary,price(formattedValue),images(DEFAULT,galleryIndex)',
           details:
-            'products/${productCode}?fields=averageRating,stock(DEFAULT),description,availableForPickup,code,url,price(DEFAULT),numberOfReviews,manufacturer,categories(FULL),priceRange,multidimensional,tags,images(FULL)',
+            'products/${productCode}?fields=averageRating,stock(DEFAULT),description,availableForPickup,code,url,price(DEFAULT),numberOfReviews,manufacturer,categories(FULL),priceRange,multidimensional,tags,images(FULL),bundleTemplates',
```

Comment:
Can we introduce a separate `bundleTemplates` product loading scope config? We can provide the default config for it in the bundles lib. See docs https://sap.github.io/spartacus-docs/loading-scopes/
```
bundleTemplates: 'products/${productCode}?fields?=bundleTemplates'
```

And in your UI component call:
`ProductService.get(productCode, ProductScope.BUNDLE_TEMPLATES)`
(You'll might to [augment](https://sap.github.io/spartacus-docs/type-augmentation/) the `ProductScope` model in your bundle lib)